src/views/profilePage/EditProfile.jsx

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable jsx-a11y/no-autofocus */
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Container,
+  Row,
+  Col,
+  Image,
+  Button,
+  Form,
+  FormControl,
+  Alert
+} from 'react-bootstrap';
+import { validateUserProfile } from '../../utils/validator';
+import './profile.scss';
+import Footer from '../../components/footer/Index';
+import { editProfileRequest } from '../../redux/actions/profile';
+
+
+const EditProfile = (props) => {
+  const { editProfile, profile } = props;
+  const initialFormState = {
+    firstname: profile.firstname,
+    lastname: profile.lastname,
+    bio: profile.bio,
+  };
+  const [errors, setErrors] = useState([]);
+
+  const renderErrors = () => errors.map(error => (
+    <Alert key={error} variant="danger">{error}</Alert>
+  ));
+
+  const [formData, setFormData] = useState(initialFormState);
+
+  const cancelChanges = () => props.history.push('/userprofile');
+
+  const handleChange = (event) => {
+    const { target } = event;
+
+    setFormData({
+      ...formData,
+      [target.name]: target.value
+    });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const validationErrors = validateUserProfile(formData);
+    if (validationErrors.length > 0) {
+      return setErrors(validationErrors);
+    }
+    editProfile(formData);
+    props.history.push('/userprofile');
+  };
+
+  return (
+    <div className="profile-container">
+      <Container>
+        <Row>
+          <Col md={3}>
+            <Image className="profile-image" src="https://res.cloudinary.com/djdsxql5q/image/upload/v1547914942/ireporter/ujz7ozprk8hkyszjju7d.jpg" />
```

Comment:
@Dubby20 let's save this image URL in a variable to make the code look cleaner

Reply:
Alright Thanks.